care/facility/api/viewsets/facility_flag.py

```diff
@@ -0,0 +1,27 @@
+from django_filters import rest_framework as filters
+from rest_framework import viewsets
+
+from care.facility.api.serializers.facility_flag import FacilityFlagSerializer
+from care.facility.models import FacilityFlag
+from care.utils.custom_permissions import IsSuperUser
+
+
+class FacilityFlagFilter(filters.FilterSet):
+    flag = filters.CharFilter(field_name="flag", lookup_expr="icontains")
+    facility = filters.UUIDFilter(field_name="facility__external_id")
+
+
+class FacilityFlagViewSet(viewsets.ModelViewSet):
```

Comment:
Maybe lets also add an endpoint that returns all the available flags

Reply:
Alright